python/graphscope/client/connection.py

```diff
@@ -88,11 +90,21 @@ def delete_edges(self, edge_pks: list):
         return self._conn.batch_write(request)
 
 
+def _encode_metadata_for_authorization(username, password):
+    if username is None or password is None:
+        return None
+    secret = username + ":" + password
+    secret = base64.b64encode(secret.encode("utf-8")).decode("utf-8")
```

Comment:
1. secret -> base64? 🤣
2. `authorization` should be `authentication`
3. is there any verification for the authentication?